src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -53,6 +53,13 @@ export const CheckImg = styled.img`
     width: 147px;
   }
```

Comment:
Check the lines before line 54, they're positioning the image using position relative and one static value: `left: 266px; top: 232px`

https://github.com/Azordev/maps-project/blob/3f7926a4dd930f39cdb4a4459bee2307ed848f44/src/pages/Confirm/Confirm.styled.jsx#L48-L54

This meant that the image is not responsive to the screen size, let me show you a screenshot where the image is overflow to the bottom of my screen 

![imagen](https://user-images.githubusercontent.com/66505715/158848802-d4c2decc-b723-43f2-a75f-93221c7f4b12.png)

Please change the logic applied to another more responsive, you can use flex, grid, or you can even use `position: absolute; bottom: 10%; right: 50%;` for example. Maybe it's not the better option but it's responsive

![imagen](https://user-images.githubusercontent.com/66505715/158849527-b3039763-f48d-4342-a58a-09ebac6cdf09.png)